test/unit/mocha.spec.js

```diff
@@ -224,6 +235,11 @@ describe('Mocha', function() {
       mocha.bail();
       expect(mocha.suite._bail, 'to be', true);
     });
+
+    it('should be chainable', function() {
```

Comment:
this looks like just adding some extra coverage around the chainable methods?  unrelated to `unloadFile`, yes?

Reply:
Correct -- just noticed several chainables were missing their tests.